tests/main/ubuntu-core-gadget-config-defaults/task.yaml

```diff
@@ -116,6 +120,12 @@ execute: |
     snap get test-snapd-with-configure b|MATCH "^B$"
 
     echo "The configuration for core is applied"
-    snap get core service.rsyslog.disable|MATCH true
-    echo "And the service is masked"
-    systemctl status rsyslog.service|MATCH masked
+    snap get core "service.$SERVICE.disable" | MATCH true
+
+    if [ "$SERVICE" = ssh ]; then
+        echo "And the ssh service is disabled"
+        MATCH "SSH has been disabled by snapd system configuration" < /etc/ssh/sshd_not_to_be_run
```

Comment:
Will we need to undo this? I mean in restore: remove the file `/etc/ssh/sshd_not_to_be_run` ? Without the next intentional reboot will not have ssh anymore, no? Also we should probably check if ssh is really stopped, I get the suspicion that we don't actively stop it (and this test can now easily confirm that).

Reply:
thanks!! restore added

Reply:
> Also we should probably check if ssh is really stopped, I get the suspicion that we don't actively stop it (and this test can now easily confirm that).

this wasn't added though

Reply:
Already added